casper/src/test/scala/coop/rchain/casper/api/BlocksResponseAPITest.scala

```diff
@@ -122,7 +122,7 @@ class BlocksResponseAPITest
         genesis        <- createDagWith8Blocks(blockStore, blockDagStorage)
         blockApi       <- createBlockApi[Task](genesis.shardId, maxBlockLimit)
         blocksResponse <- blockApi.getBlocks(2)
-      } yield blocksResponse.right.get.length should be(2) // TODO: Switch to 3 when we implement block height correctly
+      } yield blocksResponse.right.get.length should be(3)
```

Comment:
This DAG is created https://github.com/rchain/rchain/blob/d45919032d2b8a307aebc35a1e3f0980ea4377de/docs/casper/images/no_fnalizable_block_mistake_with_no_disagreement_check.png
so length should be 3. Not sure how it were passing with 2, looks like there were a bug in IndexedBlockDagStorage.